src/awkward/operations/ak_from_rdataframe.py

```diff
@@ -30,14 +42,16 @@ def from_rdataframe(rdf, columns, offsets_type="int64_t", keep_order=False):
         {
             "rdf": rdf,
             "columns": columns,
+            "highlevel": highlevel,
+            "behavior": behavior,
             "offsets_type": offsets_type,
             "keep_order": keep_order,
         },
```

Comment:
The order of this dict needs to match the order of arguments, since it's used in the error message.

```suggestion
        {
            "rdf": rdf,
            "columns": columns,
            "keep_order": keep_order,
            "offsets_type": offsets_type,
            "highlevel": highlevel,
            "behavior": behavior,
        },
```